build-indices_wt/add-introns-to-gtf.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
```

Comment:
@aawdeh Would you be able to please confirm that this add-introns script has your latest changes in it? This is the script that adds intronic regions to the GTF.

Reply:
I just checked add `add-introns-to-gtf.py` and it does have the latest changes.